Cargo.toml

```
@@ -21,6 +21,7 @@ pkg-fmt = "zip"
 [dependencies]
 async-trait = "0.1.56"
 bytes = "1.1.0"
+bzip2 = "0.4.3"
```

Can you enable feature `static` so that it will be statically linked?

```suggestion
bzip2 = { version = "0.4.3", features = ["static"] }
```